files/ddraw.ini

```diff
@@ -4,7 +4,8 @@
 [Main]
 
 ;Set to one to hide areas outside map bounds when using higher than 640x480 resolution, and to zero to disable
-;HiResMode=1
+;EnableHighResolutionStencil=1
+

```

Comment:
This doesn't exist in sfall, right?  I wonder if might might thematically more belong in f2_res.ini?  (or a new f2_ce.ini: https://github.com/fallout2-ce/fallout2-ce/discussions/155)

For now, I think this is a reasonable change.